docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,35 +1,43 @@
-* Added the `ClearWanQueue` function into the `WanReplication` finalizer executor to clean the WAN Replication queue after a CR deletion (https://github.com/hazelcast/hazelcast-platform-operator/pull/403[#403])
-* Made the `LOGGING_LEVEL` configurable for Hazelcast containers (https://github.com/hazelcast/hazelcast-platform-operator/pull/352[#352])
+* Introduced High Availability Mode configuration to create clusters that are resilient to node and zone failures (https://github.com/hazelcast/hazelcast-platform-operator/pull/538[#538])
+* Added support for the Queue CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/428[#428])
+* Added support for the Cache CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/430[#430])
+* Users can enable persistence support for the Cache CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/474[#474])
+* Replaced the `autoForceStart` option with the `startupAction` that allows the user to trigger Force Start or Partial Start when applied (https://github.com/hazelcast/hazelcast-platform-operator/pull/511[#511])
+* Introduced Helm chart for Platform Operator installation
 
 == Changes
 
-* Moved scheduling of HotBackups from the HotBackup API to the CronHotBackup CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/362[#362])
-* Added a default value to the `kind` property in the `WanReplication` CR specification (https://github.com/hazelcast/hazelcast-platform-operator/pull/374[#374])
-* Added a namespaced Role and RoleBinding that are created and passed to the namespaced service account of the Hazelcast cluster to grant secret access (https://github.com/hazelcast/hazelcast-platform-operator/pull/407[#407])
-* Replaced the REST client with Go for triggering backups (https://github.com/hazelcast/hazelcast-platform-operator/pull/381[#381])
-* Changed the default repository to Hazelcast Enterprise when the `licenseKeySecret` is present (https://github.com/hazelcast/hazelcast-platform-operator/pull/402[#402])
-* Enabled TLS for the backup server sidecar (https://github.com/hazelcast/hazelcast-platform-operator/pull/350[#350])
-* Added a retry for backup service calls (https://github.com/hazelcast/hazelcast-platform-operator/pull/434[#434])
-* Added `WanReplication` to the API reference (https://github.com/hazelcast/hazelcast-platform-operator/pull/355[#355])
-* Refactored the Multimap and Topic CR data structures and Hazelcast's `configMap` creation (https://github.com/hazelcast/hazelcast-platform-operator/pull/393[#393])
-* Upgraded Hazelcast and Management Center versions to 5.1.4
+* Added connectivity check to WAN Replication CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/541[#541])
+** When creating a WAN Replication CR, a user will be sure that the target cluster is reachable
+* Persist Wan Replication configuration into generated Hazelcast ConfigMap (https://github.com/hazelcast/hazelcast-platform-operator/pull/513[#513])
```

Comment:
```suggestion
* Added persistence for WAN Replication configurations. WAN Replication configurations are now stored in the Hazelcast ConfigMap so that when member containers restart, replication continues (https://github.com/hazelcast/hazelcast-platform-operator/pull/513[#513])
```